app/styles/ember-rdfa-editor/_c-annotation-content-nl.scss

```diff
@@ -189,6 +196,26 @@
     content: 'Nummer';
   }
 
+  [property="ext:hasParagraph"]:before {
+    content: 'Paragraaf';
+  }
+
+  [resource="http://mu.semte.ch/vocabularies/ext/Title"] + [property="say:heading"]:before {
+    content: 'Titel';
+  }
+
+  [resource="http://mu.semte.ch/vocabularies/ext/Chapter"] + [property="say:heading"]:before {
+    content: 'Hoofdstuk';
+  }
+
+  [resource="http://mu.semte.ch/vocabularies/ext/Section"] + [property="say:heading"]:before {
+    content: 'Afdeling';
```

Comment:
Isn't this an odd translation of "section"?

Reply:
That's the legal name for it that I have to follow.